pkg/cmd/scan/output/console.go

```diff
@@ -98,7 +98,7 @@ func (c *Console) Write(analysis *analyser.Analysis) error {
 					if change.Type == diff.UPDATE {
 						if change.JsonString {
 							prefix := "           "
-							fmt.Printf("%s%s\n%s%s\n", whiteSpace, pref, prefix, jsonDiff(change.From, change.To, prefix))
+							fmt.Printf("%s%s\n%s%s\n", whiteSpace, pref, prefix, jsonDiff(change.From, change.To, true))
```

Comment:
Maybe we can pass `isatty.IsTerminal(os.Stdout.Fd())` instead of true here, and thus in `jsonDiff` we only need to pass the `coloring` parameter instead of doing an `&&` WDYT ?